azavea/temperate
# Add support for Amazon ECS deployment

## Overview

Add all of the AWS/Terraform resources necessary to support running the application (only over HTTPS for now) inside of the core infrastructure shared with the Climate API. In addition, update `ALLOWED_HOSTS` inside of the Django settings so the ALB host header is added when the application boots.

Also, wire up Rollbar (with a different token than the Climate API) and Papertrail (with different log collectors). Lastly, add Watchman based health checks (only checks database connectivity).

## Testing Instructions

- Test the health check by hitting the following URL: https://planit.staging.climate.azavea.com/health-check/
- Test Rollbar by inspecting the following item: https://rollbar.com/Azavea/ClimateChange/items/70/
- Test Papertrail integration by browsing the `Staging (PlanIt)` log collector output: https://papertrailapp.com/groups/6464521/events

Testing the rest of the application is a little more difficult because no real routes exist yet. But the fact that I was able to run migrations successfully via the `StagingManagementPlanItApp` task gives me confidence that the Django configuration is functional.

Closes https://github.com/azavea/climate-planit/issues/22
